docker/build.gradle

```diff
@@ -1,53 +1,381 @@
+plugins {
+    id 'com.bmuschko.docker-remote-api' version '9.3.1'
+     id 'idea'
+}
+
+
+//import org.gradle.nativeplatform.platform
+import org.gradle.internal.os.OperatingSystem
+
+// Import task types
+import com.bmuschko.gradle.docker.tasks.image.*
+
+
 apply plugin: 'idea'
 
 idea.module {
     excludeDirs += file("data")
     excludeDirs += file("logs")
 }
 
-task("buildEhrcron", group: "Docker", type: Exec, description: "Generates the ehrcron docker image") {
+//Registering credential to use the bmuschko plugin to use docker
+docker {
+    url = 'unix:///var/run/docker.sock'
+    registryCredentials {
+        url = 'https://index.docker.io/v1/'
+        username = "${dockerhubUsername}"
+        password = "${dockerhubPassword}"
+    }
+}
```

Comment:
Does this plugin know how to use the `dockertokenpath` you're using for the `dockerLogin` task? The build is currently failing because `dockerhubPassword` is not defined.

Reply:
this plugin uses the username and the password